tests/app/va/va_profile/test_va_profile_client_for_profile_v3.py

```diff
@@ -209,6 +294,52 @@ def test_ut_get_telephone_raises_NoContactInfoException_if_no_mobile_telephones_
         with pytest.raises(NoContactInfoException):
             mock_va_profile_client.get_telephone_with_permission(recipient_identifier, sample_notification())
 
+    def test_ut_get_telephone_raises_NoContactInfoException_if_number_classified_as_not_mobile(
+        self, rmock, mock_va_profile_client, mock_response, recipient_identifier, url, mocker, sample_notification
+    ):
+        mock_feature_flag(mocker, FeatureFlag.VA_PROFILE_V3_IDENTIFY_MOBILE_TELEPHONE_NUMBERS, 'True')
+
+        telephones = mock_response['profile']['contactInformation']['telephones']
+        for telephone in telephones:
+            telephone['classification'] = {'classificationCode': 1}
+        mock_response['profile']['contactInformation']['telephones'] = telephones
+        rmock.post(url, json=mock_response, status_code=200)
+
+        with pytest.raises(NoContactInfoException):
+            mock_va_profile_client.get_telephone_with_permission(recipient_identifier, sample_notification())
+
+    def test_ut_get_telephone_with_permission_prefers_user_specified_mobile_phone(
```

Comment:
```suggestion
    def test_get_telephone_with_permission_prefers_user_specified_mobile_phone(
```
We got rid of the test type prefix.